master/internal/core_experiment.go

```diff
@@ -88,6 +88,37 @@ func (m *Master) getExperimentCheckpointsToGC(c echo.Context) (interface{}, erro
 		args.ExperimentID, args.ExperimentBest, args.TrialBest, args.TrialLatest)
 }
 
+func (m *Master) getExperimentModelFile(c echo.Context) error {
```

Comment:
I see we are using the old echo server to download the file.  Is that because the grpc server doesn't let us return the plain content in a browser-friendly fashion?

Reply:
I’m not sure how to pass non-json object through proto… but we might be able to trigger download from front end with json object, but for now, can we keep it in echo server